src/app/App.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
-//import { useNavigate } from 'react-router';
 import AddDestination from './pages/AddDestination/AddDestination';
 import DestinationDetailView from './pages/DestinationDetailView/DestinationDetailView';
 import MapView from './pages/MapView/MapView';
+import Itinerary from './pages/Itinerary/Itinerary';
 
 function App(): JSX.Element {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path="/" element={<AddDestination />} />
+        <Route path="/" element={<Itinerary />} />
+        <Route path="/addDestination" element={<AddDestination />} />
         <Route
```

Comment:
looks like this is twice for `path="/"`?

Reply:
I changed the path="/". The first line (red) is the old value, the second line (green) is the new value.